.github/workflows/pytorch-version-tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       max-parallel: 10
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8]
```

Comment:
Should we add python 3.9 here ?

Reply:
We should then exclude older pytorch version that were not delivered for python 3.9

Reply:
@vfdev-5 should we add python 3.9 and exclude PyTorch old version?

Reply:
@bibhabasumohapatra let's try that

Reply:
@vfdev-5  no need for PyTorch and 3.9 update in this PR right?

Reply:
> We should then exclude older pytorch version that were not delivered for python 3.9

yes like we did here:
https://github.com/pytorch/ignite/blob/94bf2dab857f86e3045237013d94a8bde4e164cf/.github/workflows/pytorch-version-tests.yml#L18-L20

we just need to know the exact versions

Reply:
@bibhabasumohapatra we can include that in this PR